homeassistant/components/hassio/__init__.py

```diff
@@ -110,6 +124,7 @@
     SERVICE_ADDON_START: ("/addons/{addon}/start", SCHEMA_ADDON, 60, False),
     SERVICE_ADDON_STOP: ("/addons/{addon}/stop", SCHEMA_ADDON, 60, False),
     SERVICE_ADDON_RESTART: ("/addons/{addon}/restart", SCHEMA_ADDON, 60, False),
+    SERVICE_ADDON_UPDATE: ("/addons/{addon}/update", SCHEMA_ADDON, 60, False),
```

Comment:
I'm not sure it makes sense to have a service to update an addon.
Do we really want folks automating this?
They will do that without reading release notes, and then complain to the maintainer of that add-on ~~if~~ when it breaks.

I guess with this, we could deprecate the auto-update feature inside the supervisor.

Reply:
> I guess with this, we could deprecate the auto-update feature inside the supervisor.

👆 My thought as well.

Reply:
I could update the description in the schema to have a cautionary note to try to head off complaints

Reply:
@frenck @ludeeus ?

Reply:
Since we can use this to sunset the auto feature in the supervisor, I'm in favor of adding it, description should warn